platform-sdk/swirlds-crypto-tss/src/main/java/com/swirlds/crypto/signaturescheme/api/PairingPrivateKey.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.crypto.signaturescheme.api;
+
+/**
+ *  A Mock PairingPrivateKey
+ */
+public record PairingPrivateKey() {}
```

Comment:
If these are Mocks, do they need to implement an interface?  be moved to a mock package?  I'm very confused by empty records with real sounding class names but no interfaces they correspond to or a sense of what the real / non-mock content would look like.

Reply:
If this intended to be a place holder for further expansion of adding real data, I wouldn't call it a Mock.

Reply:
We agreed on "prototype".
Changed